transform/mattermost-analytics/models/staging/cws/_cws__models.yml

```diff
@@ -64,3 +64,23 @@ models:
       - name: is_terms_accepted
         description: Whether the user accepted the terms and conditions.
 
+  - name: stg_cws__marketing
+    description: |
+      CWS Marketing data.
+
+    columns:
+      - name: marketing_id
+        description: Unique Identifier for Marketing data.
+        tests:
+          - not_null
+      - name: email
+        description: The user's email
+        tags: ['pii']
+      - name: subscribed_content
+        description: The content the user subscribed to.
+      - name: server_id
```

Comment:
Tests should be added here as well.

Reply:
Should we use validate_email macro in the model or `not_null` and `dbt_utils.not_empty_string` would be sufficient?

Reply:
Added `not_null` test here.